lib/prompts/sandboxesPrompt.ts

```diff
@@ -70,13 +71,19 @@ export async function sandboxTypePrompt(): Promise<SandboxTypePromptResponse> {
 
 export function deleteSandboxPrompt(
   promptParentAccount = false
-): Promise<DeleteSandboxPromptResponse> | void {
+): Promise<DeleteSandboxPromptResponse> {
   const accountsList = getConfigAccounts();
   const choices = promptParentAccount
     ? mapNonSandboxAccountChoices(accountsList)
     : mapSandboxAccountChoices(accountsList);
   if (!choices.length) {
-    return;
+    return Promise.reject(
```

Comment:
I like the thought here to handle this case better, but you may be able to do this in a slightly simpler way. Throwing from here means that we'd need to wrap the call to `deleteSandboxPrompt` in a try/catch. It might be easier to return void here and catch this case with an `if ! parentAccountPrompt.account` somewhere around [here](https://github.com/HubSpot/hubspot-cli/pull/1383/files#diff-2e9c34f9182dc290b80774109b85c7614b0d947cf32d90028639bcec586ed947R93).